controller/controller_server/csi_controller_server.py

```diff
@@ -343,9 +343,53 @@ def CreateSnapshot(self, request, context):
             return csi_pb2.CreateSnapshotResponse()
 
     def DeleteSnapshot(self, request, context):
-        # TODO
-        logger.info("Delete snapshot")
-        return csi_pb2.DeleteSnapshotResponse()
+        set_current_thread_name(request.snapshot_id)
+        logger.info("DeleteSnapshot")
```

Comment:
better "DelateSnapshot start"

Reply:
should I add a "start" to the rest of functions as well?

Reply:
If other methods use this way ignore it